kitsune/sumo/static/sumo/js/analytics.js

```diff
@@ -1,119 +1,56 @@
-      results.push(this.split(/\s*\|\s*/));
-    });
-    return results;
-  }
-
-  // Track clicks to links with data-ga-click attr.
-  $('body').on('click', 'a[data-ga-click]', function(ev) {
+(function($) {
+  'use strict';
+  // Track clicks to form buttons with data-event-category attr.
+  $('body').on('click', 'button[data-event-category]', function(ev) {
     ev.preventDefault();
 
     var $this = $(this);
-    var gaData = parseAnalyticsData($this.data('ga-click'));
-    var href = $this.attr('href');
+    trackEvent(
+      $this.attr('data-event-category'),
+      $this.attr('data-event-action'),
+      $this.attr('data-event-value'),
+      $this.attr('data-event-label')
+    );
 
-    $(gaData).each(function() {
-      _gaq.push(this);
-    });
-
-    // Delay the click navigation by 250ms to ensure the event is tracked.
+    // Delay the form post by 250ms to ensure the event is tracked.
     setTimeout(function() {
-      document.location.href = href;
+      $this.closest('form').submit();
     }, 250);
 
     return false;
   });
 
-  // Track clicks to form buttons with data-ga-click attr.
-  $('body').on('click', 'button[data-ga-click]', function(ev) {
+  // Track clicks to links with data-event-category attr.
+  $('body').on('click', 'a[data-event-category]', function(ev) {
     ev.preventDefault();
 
     var $this = $(this);
-    var gaData = parseAnalyticsData($this.data('ga-click'));
+    var href = $this.attr('href');
 
-    $(gaData).each(function() {
-      _gaq.push(this);
-    });
+    trackEvent(
+      $this.attr('data-event-category'),
+      $this.attr('data-event-action'),
+      $this.attr('data-event-value'),
+      $this.attr('data-event-label')
+    );
 
-    // Delay the form post by 250ms to ensure the event is tracked.
+    // Delay the click navigation by 250ms to ensure the event is tracked.
     setTimeout(function() {
-      $this.closest('form').submit();
+      document.location.href = href;
     }, 250);
 
     return false;
   });
-
-
-  // Track reads (5secs and 10secs on page) of product landing pages.
-  if ($('body').is('.product-landing')) {
-    setTimeout(function() {
-      if (_gaq) {
-        _gaq.push(['_trackEvent',
-                   'Landing Page Read - 10 seconds',
-                   $('body').data('product-slug')]);
-      }
-    }, 10000);
-
-    setTimeout(function() {
-      if (_gaq) {
-        _gaq.push(['_trackEvent',
-                   'Landing Page Read - 5 seconds',
-                   $('body').data('product-slug')]);
-      }
-    }, 5000);
-  }
-}
-
-
-function trackEvent(category, action, value) {
-  if (_gaq) {
-    _gaq.push(['_trackEvent', category, action, value]);
-  }
-}
-
-function trackPageview(value) {
-  if (_gaq) {
-    _gaq.push(['_trackPageview', value]);
+})(jQuery);
+
+function trackEvent(category, action, value, label) {
+  if (window.gtag) {
+    window.gtag('event', action, {
+      'event_category': category,
+      'value': value,
```

Comment:
Are we sure this should be `value` and not `event_value`? I know the doc specifies `value`, but might be worth double checking since we're having trouble seeing this in reports.